caproto/ioc_examples/thermo_sim.py

```diff
@@ -33,32 +33,32 @@ def __init__(self, *args, period, **kwargs):
         self._T0 = time.monotonic()
         self.period = period
 
-    readback = pvproperty(value=[0], dtype=float, read_only=True)
+    I = pvproperty(value=[0], dtype=float, read_only=True)
```

Comment:
You could have also specified `name="SP"` and left it as `setpoint` for the property name ([ref](https://github.com/NSLS-II/caproto/blob/d1610fa259cb7ff83b44c5c7eed1044faa48eef1/caproto/server/server.py#L288))

Reply:
:+1: for that approach, to keep the names readable by humans.

Reply:
change has been made !-)